msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -336,6 +336,26 @@ The query parameters are:
 > [!NOTE]
 > You can see `channelId` and `groupId` in the URL from the channel.
 
+### Generate deep links to chat messages
+
+Use this deep link format to go to a message in personal and group chats in Teams:
```

Comment:
Use this deep link format to navigate a user to a message in a personal or group chat in Teams.

Reply:
updated